utils/rudderstack_initialize.ts

```diff
@@ -1,90 +1,77 @@
 export interface RudderstackClientSideEvents {
-  identify: (
-    userId: string,
-    name: string,
-    email: string,
-    anonymousId: string
-  ) => void;
+  identify: (userId: string, name: string, email: string, anonymousId: string) => void
   track: (
     userId: string,
     event: string,
     properties: { [key: string]: any },
     anonymousId: string
-  ) => void;
-  page: (
-    type: string,
-    pageName: string,
-    properties: { [key: string]: any }
-  ) => void;
+  ) => void
+  page: (type: string, pageName: string, properties: { [key: string]: any }) => void
 }
 
-let rudderAnalyticsInstance: typeof import('rudder-sdk-js') | null = null;
```

Comment:
@MuskanPaliwal This `rudderAnalyticsInstance` is being removed in this PR. Is this fine?

Reply:
Haan it should be. But I this was needed because of the `null` type